specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2020-01-01/DataEncryptionKeys.json

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
Why are there multiple copies of this API, are the APIs actually different?  Note that you cna include a swagger spec in multiple tags - there should be no reason to copy this file multiple times for different api-versions.  This is only necessary if they are actually different.